services/UserService.ts

```diff
@@ -0,0 +1,21 @@
+import { Service } from 'typedi';
+import { Repositories, TransactionsManager } from '../repositories';
+import { UserModel } from '../models/UserModel';
+import { Uid } from '../types/Internal';
+
+@Service()
+export class UserService {
+  private transactionsManager: TransactionsManager;
+
+  constructor(transactionsManager: TransactionsManager) {
+    this.transactionsManager = transactionsManager;
```

Comment:
Will all the services share a TransactionsManager? Or is it possible to put the DataSource into the constructor and use a new transactionsManager which would be a little closer to the membership portal currently.

Reply:
Yes, all services will be injected with the same TransactionsManager. It is also possible to take the DataSource as an argument in the constructor and create a new TransactionsManager each time. I don't really have a strong opinion on this so if you think it would be better to match the membership portal I can change it.